web-console/src/app/(root)/(authenticated)/(app)/streaming/inspection/page.tsx

```diff
@@ -85,7 +85,13 @@ const TableInspector = ({
   const logError = (error: Error) => {
     console.error('InspectionTable error: ', error)
   }
-  const [rows, setRows] = useState<Row[]>([])
+  const [relationsRows, setRelationsRows] = useState<Record<string, Row[]>>({})
+  const rows = relationsRows[caseIndependentName] ?? []
+  const setRows = (rows: SetStateAction<Row[]>) =>
+    setRelationsRows(old => ({
+      ...old,
+      [caseIndependentName]: rows instanceof Function ? rows(old[caseIndependentName] ?? []) : rows
```

Comment:
so `relationRows` can only grow over time? seems odd

Reply:
Yes, but only until you leave the page, when component gets unmounted (/streaming/inspection page is exited) and the state dropped

Reply:
The previous list of rows gets overwritten with a new list of rows on update